src/AutorestSwift/Code Generator/AutoRestPlugin.swift

```diff
@@ -3,25 +3,44 @@ import NIO
 import os.log
 
 class AutoRestPlugin {
-    public func start() throws -> EventLoopFuture<AutoRestPlugin> {
-        let logger = Logger(withFileName: "autorest-swift-debug.log")
-
-        let autoRestHandler = AutoRestHandler()
-        logger.logToURL("Start NIO pipe")
+    var logger: Logger!
+    var sessionId: String?
 
-        let channel = NIOPipeBootstrap(group: MultiThreadedEventLoopGroup(numberOfThreads: 5))
+    public func start() throws -> EventLoopFuture<AutoRestPlugin> {
+        logger = Logger(withFileName: "autorest-swift-debug.log")
+        logger.logToURL("Starting NIO pipeline")
+        let handler = AutoRestHandler(withPlugin: self)
+        let channel = NIOPipeBootstrap(group: MultiThreadedEventLoopGroup(numberOfThreads: 1))
```

Comment:
Made this one thread so logging statements don't get interleaved. This should make comprehending the logs easier.